contracts/mocks/WhitelistMock.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.4.8;
```

Comment:
Was using 0.4.8 intentional here?

Reply:
Not really. Good catch!
Should I use `pragma solidity ^0.4.18;`? I found most contracts here use this pragma, however some contracts are different. Do you have any guidelines for the compiler version?

Reply:
Yes please, use .18 :)

Reply:
Done